home/views.py

```diff
@@ -19,9 +22,10 @@ def index(request):
     context['google_analytics_key'] = settings.GOOGLE_ANALYTICS_KEY
 
     if settings.GOOGLE_ANALYTICS_API:
-         # Based on https://developers.google.com/analytics/devguides/reporting/core/v3/quickstart/service-py
-        from apiclient.discovery import build
-        from oauth2client.service_account import ServiceAccountCredentials
+        # Based on https://developers.google.com/analytics/devguides/reporting/core/v3/quickstart/service-py
+        # What in the HELL! were these two imports doing here?
```

Comment:
Comment can (imho) be removed together with the imports. 
My best guess is that the import was put there to be prevent the import when the GAA setting is not present.

Reply:
Yes. You're right.
Just noticed.
Agree with the comments.